tests/models/profile.test.js

```diff
@@ -0,0 +1,22 @@
+import { expect } from 'chai';
+import { sequelize, dataTypes, checkModelName, checkPropertyExists}  from 'sequelize-test-helpers';
+import ProfileModel from '../../server/models/profile';
+import UserModel from '../../server/models/user';
+
+describe('Profile properties', () => {
+  const Profile = ProfileModel(sequelize, dataTypes);
+  const profile = new Profile();
+  checkModelName(Profile)('Profile');
+  context('properties', () => {
+       ['gender','twitterUrl','role','imageurl','facebookUrl','bio','dateOfBirth'].forEach(checkPropertyExists(profile));
```

Comment:
nice block of code.
but I think using it() makes it more readable

Reply:
That would require writing the same line of code 9 times which i think is verbose.
```
has property gender
```
is descriptive enough.